integration_tests/test_mesh_lens_correction.py

```diff
@@ -155,6 +147,8 @@ def test_coarse_mesh_failure(
     example_for_input['rerun_pointmatch'] = False
     outjson = os.path.join(output_directory, 'mesh_lens_out0.json')
 
+    print(json.dumps(example_for_input, indent=2))
```

Comment:
can we change this to just json.dumps rather than a print statement. unless you require something to be printed

Reply:
@mahaling can you please finish your review?